sklearn/metrics/_ranking.py

```diff
@@ -145,6 +145,8 @@ def average_precision_score(
         Target scores, can either be probability estimates of the positive
         class, confidence values, or non-thresholded measure of decisions
         (as returned by :term:`decision_function` on some classifiers).
+        For y_scores from "decision_function", positive scores are required to
+        correspond to the positive class.
```

Comment:
Lets link to `decision_function` in the glossary. Nitpick, what do you two think of something like:

```suggestion
        For :term:`decision_function` scores, values strictly greater than zero should
        indicate the positive class.
```

Reply:
Sounds almost perfect to me :-). A decision function of zero would correspond to a probability of 0.5. To my understanding, 0.5 would be assigned to the positive class. Please correct me if I'm wrong. So I changed it to
`For :term:`decision_function` scores, values greater than or equal zero should
        indicate the positive class.`

Reply:
Really not important, just FYI: there is this discussion going on for hyperplanes of SVMs (https://www.researchgate.net/post/In_SVM_what_happens_when_test_data_has_a_point_on_the_hyperplane_How_does_it_categorise_it). 

If I understand sklearn correctly, I think datapoints in any hyperplane would also be assigned to the positive class. So >= 0 would correspond to the positive class.